iterators.tex

```diff
@@ -276,7 +276,7 @@
   template <class I>
   concept bool Readable@\newtxt{() \{}\oldtxt{ =}@
     @\newtxt{return}@ Semiregular<I>@\newtxt{()}@ &&
-      requires (I i) {
+      requires (@\newtxt{const }@I i) {
```

Comment:
...except that those overloads cannot have "equivalent semantics", of course. I'll make this modifying, and introduce a non-modifying `operator*` in `WeakInputIterator`.